samples/advanced/custom-capabilities-filter_mca.html

```diff
@@ -0,0 +1,117 @@
+<!doctype html>
+<html lang="en">
```

Comment:
Please keep the filename syntax for this sample:

`custom-capabilities-filter-mca` instead of `custom-capabilities-filter_mca`